pycbc/strain/strain.py

```diff
@@ -1711,6 +1711,28 @@ def recalculate_psd(self):
         logging.info("Recalculating %s PSD, %s", self.detector, psd.dist)
         return True
 
+    def check_psd_dist(self, min_dist, max_dist):
+        """Check that the horizon distance of a detector is within a required
+        range.
+        """
+        if self.psd is None:
+            # ignore check
+            return True
+        # Note that the distance can in principle be inf or nan, e.g. if h(t)
+        # is identically zero. The check must fail in those cases.  Be careful
+        # with how the logic works out!
+        good = self.psd.dist >= min_dist and self.psd.dist <= max_dist
+        if not good:
```

Comment:
if not using good as the return parameter, then we can save a variable here by juts using the logic in the if